docs/Device.md

```diff
@@ -0,0 +1,80 @@
+# Device CLI Commands
+
+Router maintains a databse in Rocks of Devices, but the paired Console instance should always be viewed as the source of truth for a Device.
+
+> XOR Filter commands have been relocated under `router filter`.
+
+> **NOTE: Commands in these docs assume an alias for `router`.**
+
+```sh
+alias router="docker exec -it helium_router _build/default/rel/router/bin/router"
+```
+
+## Commands
+
+### List
+
+```sh
+router device all
+```
+
+Queries Console to report all devices.
+
+### Info
+
+```sh
+router device --id=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx [--stop]
+```
+
+Simple information for a device. `<id>` is the device ID from Console, not the `app_eui` or `dev_eui`.
+
+### Tracing
```

Comment:
Is tracing part of the info command?  If a separate command, there's no usage/example.  If part of the info command as implied by `--stop` mentioned below, it's confusing being within a separate subsection.